trussworks/next-graphql-fe
# Cleanup boilerplate pages

Replace index page with what was previously the "Hello" page. 
So now index page should have the backend `Hello World` message, as well as a link to Incidents. 
There is no `hello.tsx` page.

## Testing

Everything in this repo should function in exactly the same way as it did before. To test this, we'll want to run all the server and check that all the tests are still working:
1. Run the backend database and server
2. In the frontend, Run `yarn install` if you might be out of date.
3. Run `yarn dev` to serve the interface. Check that it's as described above! 
4. Back in the frontend folder, run `yarn test` to execute the complete suite of tests.
5. If everything looks good, you're done!
